### PR TITLE
Finish app tests

### LINKED ISSUE #56: Add integration testing in App.test.tsx

### LINKED ISSUE #63: Collapse menu after navItem is clicked
Currently, the user can display on the navbar and expand the menu and navigate the application. 

The issue is that the menu bar should collapse when the user clicks on the nav item and hide the menu.

### LINKED ISSUE #80: Finalize CI to deploy

### PR DESCRIPTION
**Pull request by BlakeDonn**
### What does this PR do?

* Adds final integration testing for app
* Adjusts onClick of navbar elements to close dropdown menu when clicked
* Adjusts replaceAll in report to replace 
* Adds alt text to home picture
* Adds some deployment refactoring

### Where should the reviewer start?

* Start by checking CI is passing or not

### How should this be manually tested?

* Pull down the branch and check for the described changes above

### Any background context you want to provide?

* Hit a pretty hard wall with animation testing, used some workarounds

### What are the relevant tickets?
closes #63 
closes #56 
closes #80

### CODE REVIEW

File: src/components/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -43,18 +46,21 @@ const DropdownMenu = () => {
...
+        >Journey</DropdownItem>
       </Link>
 
       <DropdownItem data-testid='to-login'>Login</DropdownItem>
 
-      {userReport && <Link to="/report"  data-testid='to-login'>
-      <DropdownItem>Report</DropdownItem>
+      {userReport && <Link to="/report"  data-testid='to-login' onClick={props.setOpen}>
+        <DropdownItem 
+      >Report</DropdownItem>
```
Comment (Atos20) [inline src/components/NavBar/NavBar.tsx:63]: awesome addition thank BLake

File: .travis.yml [hunk 2]
```diff
@@ -9,3 +9,5 @@ deploy:
     secure: pR3QCS2dvgxCTgc1NFnXDJxw5hyh87kCUICKty4qpqOw5bojwK852v8ALyVqfWS/PNnikW7mRUIvYvNLtHZpKq4NkUTrQUjeoL1uEFV+ERxId5aDkBjuv88d92M/khsf/utvGFOxTsWqsLViW92J4uHZ/Zfb9vbrvNNAMDlqnShuwJZwNBAUqAZLJiHnIC+1pDxh88eBXAmnbU6bs+xKN2GzLZjA/GgPBl1DPPE6S2AL1a2PPjwFhwFV7jGgTzfTaPPuUdpOP4KSde0v9LIYlDvtVGwZuUby79FDhQrUiVLZ/nLlSqguYeja2a4PJcb7Dw8+DjgJcA1LvFqqxy72GXEf85lJw//9vrvsow3Fk9ofrmYpdpW1tnAZsYZ3SBpTeKcWv36HRU8aV7zUUZaoOTGcO+2GOrxgHZrSK2Ek8kbDQ6SVnoS8+TsDLHDHSn+pKE0lH+ZxxuiWkdBs2lq9OqAKqpEliv2bRoKZ15i9+63UeGOAKLukbCjKLOAj8ZeJNjd7j0pyC6XENfycgqSPebNi1pkIl2eXixqVWgF7JpAfEBZTllrRURsejWVSuPR4B8wPhMyZU2MlgmK6mR4qiaHF3SsRSj5C9g6Fu69L7VOuhpCi6jjsbybOjesls5vfMBlYMKAE9YAgvLGMI5N4eKNPCf68gx1iOZhzcToTI6I=
   app: dream-home-cap
   skip_cleanup: true
+  on:
```
Comment (Atos20) [inline .travis.yml:11]: what is this doing?

File: .travis.yml [hunk 3]
```diff
@@ -10,3 +10,4 @@ deploy:
   app: dream-home-cap
   on:
     all_branches: true
+    skip_cleanup: true
```
Comment (Atos20) [inline .travis.yml:13]: what is the clean up for?

**Review by timkeresey — COMMENTED** [review]
Great! We may need to update the API call tests once those are properly linked up.

**Review by Atos20 — APPROVED** [review]

**Pull request merged**